Gopkg.toml

```diff
@@ -85,3 +85,7 @@ required = [
 [[constraint]]
   branch = "master"
   name = "github.com/golang/glog"
```

Comment:
Does `klog` replaces `glog` completely? If so, this can be removed.

Reply:
It's actually still used in some of the tests. Was going to fix that in another PR.